src/components/client/slider/slider.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Swiper, SwiperSlide } from 'swiper/react'
+import 'swiper/css'
+import 'swiper/css/navigation'
+import 'swiper/css/pagination'
+import { Autoplay, Navigation, Pagination } from 'swiper/modules'
+import Image from 'next/image'
+import Slide from './slide'
+const Slider: React.FC = () => {
+  return (
+    <>
+      <div className="mx-auto mb-12 mt-10 max-w-6xl">
```

Comment:
hero section have bead adapted to mobile, delete hero and adaptive this component to 375

Reply:
Adapted without hero section